redi-connect-backend/lib/email/email.js

```diff
@@ -364,20 +364,24 @@ const sendMenteeRequestAppointmentEmail = ({
   }).subscribe()
 }
 
-const sendMentorPendingReviewAcceptedEmailTemplate = fs.readFileSync(path.resolve(__dirname, 'templates', 'welcome-to-redi.mjml'), 'utf-8')
-const sendMentorPendingReviewAcceptedEmailParsed = mjml2html(sendMentorPendingReviewAcceptedEmailTemplate, {
-  filePath: path.resolve(__dirname, 'templates')
-})
+const convertTemplateToHtml = (rediLocation, templateString) => {
+  const convertTemplate = fs.readFileSync(path.resolve(__dirname, 'templates', `${templateString}-${rediLocation.toLowerCase()}.mjml`), 'utf-8')
```

Comment:
Nice! I feel this is a good example of [clean code guidelines regarding functions](https://github.com/ryanmcdermott/clean-code-javascript#functions) - keep it up!